packages/block-editor/package.json

```diff
@@ -36,6 +36,7 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
 		"@wordpress/keycodes": "file:../keycodes",
+		"@wordpress/nux": "file:../nux",
```

Comment:
Not sure that it's a good idea to have `block-editor` depend on the `nux` package. I was thinking that we could have `block-editor` define some `Slot`s which `edit-post` fills in. Thoughts?

Reply:
Yeah, I'm not sure tips make sense in the raw form of a block editor.

`Slots` sound like a solution, though I'm not sure what the wider use case for those slots would be. Maybe making them experimental slots to deter usage is the way forward.

Reply:
What do you think, @youknowriad?

Reply:
I think it's fine to have the dependency if we have some sort of editor setting saying that tips are disabled by default?

Other question: I assume this is because of the inline tips? What's the difference between that component and just a notice component? (Trying to understand why we need the dependency)?

Reply:
> Other question: I assume this is because of the inline tips? What's the difference between that component and just a notice component? (Trying to understand why we need the dependency)?

It's so that we have access to the `core/nux` data store which lets us implement the _Disable all tips_ functionality.

Reply:
> I think it's fine to have the dependency if we have some sort of editor setting saying that tips are disabled by default?

Did this in 48652f9!